src/main/java/io/javalin/embeddedserver/jetty/websocket/WebSocketHandler.java

```diff
@@ -42,34 +51,67 @@ public void onError(@NotNull ErrorHandler errorHandler) {
         this.errorHandler = errorHandler;
     }
 
+    public Collection<WsSession> getSessions() {
```

Comment:
Set?

Reply:
Changed. I generally code to the most abstract interface... but I don't really have a strong argument for doing that.